src/pages/orders/components/OrderDetailsCard/OrderDetailsCardFooter/OrderDetailsCardFooter.tsx

```diff
@@ -229,14 +230,15 @@ const OrderDetailsCardFooter = ({ sendFile }: { sendFile: (file: File) => void }
             {!!isModalOpenFor('EmailVerificationModal') && (
                 <EmailVerificationModal
                     isModalOpen
-                    nextRequestTime={verificationNextRequest!}
+                    nextRequestTime={verificationTokenExpiry!}
```

Comment:
is the assertion necessary?

Reply:
yup, i know for when we show this modal, there will be data there, but because the typing says it can be undefined, this has assertion